src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -119,7 +119,7 @@ void fillInnerParts() {
         employeeListPanel = new EmployeeListPanel(logic.getFilteredEmployeeList());
         employeeListPanelPlaceholder.getChildren().add(employeeListPanel.getRoot());
 
-        projectListPanel = new ProjectListPanel(logic.getFilteredEmployeeList()); // to be changed to project list
+        projectListPanel = new ProjectListPanel(logic.getFilteredProjectList()); // to be changed to project list
```

Comment:
Can the comment be removed? I believe it is already project list.

Reply:
Yup I will remove the unnecessary comments